CHANGES.rst

```diff
@@ -20,6 +20,8 @@ dev (master)
 
 * ... [Short description of non-trivial change.] (Issue #)
 
+* Timeout can no longer be set as boolean, and must be greater than zero. 
+  (PR #924)
```

Comment:
Err... not sure how that happened, must have gotten too excited with hotkeys. Fixed now.